src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -183,6 +185,10 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             // Checks are in the LanguageParser unless otherwise noted.
             switch (feature)
             {
+                // C# 7.1 features.
+                case MessageID.IDS_DefaultInterfaceImplementation:
```

Comment:
This is not correct; this feature is targeting 8.0 or later per LDM. Probably a PROTOTYPE comment is good enough.